.github/workflows/docker-slim.yml

```diff
@@ -14,30 +14,30 @@ on:
 
 jobs:
   build:
+    # https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#running-a-workflow-based-on-the-conclusion-of-another-workflow
+    if: ${{ github.event.workflow_run.conclusion == 'success' }} and github.repository_owner == 'alshedivat'
     runs-on: ubuntu-latest
-    if: github.repository_owner == 'alshedivat'
-
     defaults:
       run:
         working-directory: ${{ github.workspace }}
 
     steps:
       - name: Checkout
-        uses: actions/checkout@v3
+        uses: actions/checkout@v4
 
       - name: Login
-        uses: docker/login-action@v2
+        uses: docker/login-action@v3
         with:
           username: ${{ secrets.DOCKER_USERNAME }}
           password: ${{ secrets.DOCKER_PASSWORD }}
 
       - name: update docker-compose
         shell: bash
         run: |
-          sed -i "s|\.:|${{ github.workspace }}:|g" ${{ github.workspace }}/docker-compose.yml 
+          sed -i "s|\.:|${{ github.workspace }}:|g" ${{ github.workspace }}/docker-compose.yml
           cat ${{ github.workspace }}/docker-compose.yml
 
-      - uses: kitabisa/docker-slim-action@v1.0.3
+      - uses: kitabisa/docker-slim-action@v1
```

Comment:
Why this version is being downgraded? 

I have discussed this with the developer and version 1 would definitely cause problems.

Reply:
Actually I did this because I assumed that if we only setup the major version, it would automatically use the latest minor version, but I couldn't find anything that backs my assumption. Reverting this change.